### PR TITLE
feat(TDP-2802): folders

### PR DESCRIPTION
**Pull request by jsomsanith**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-2802

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)
Add folder list + remove and click actions

**(Optional) Other information**:

### CODE REVIEW

File: dataprep-webapp/src/app/settings/actions/preparation-actions-service.js [hunk 1]
```diff
@@ -23,6 +23,18 @@ export default class PreparationActionsService {
 		this.TalendConfirmService = TalendConfirmService;
 	}
 
+	refreshCurrentFolder() {
+		this.FolderService.refresh(this.state.inventory.folder.metadata.id);
+	}
+
+	popRemoveSuccess(preparation) {
```
Comment (frassinier) [inline dataprep-webapp/src/app/settings/actions/preparation-actions-service.js]: What does `pop` stand for?

Comment (jsomsanith) [inline dataprep-webapp/src/app/settings/actions/preparation-actions-service.js]: What are you doing here at 22:48 first ?

Comment (jsomsanith) [inline dataprep-webapp/src/app/settings/actions/preparation-actions-service.js]: Pop like `pop a toaster` ?
I can rename it if needed

File: dataprep-webapp/src/app/services/folder/folder-service.js [hunk 2]
```diff
@@ -117,6 +140,17 @@ export default function FolderService($q, state, StateService, FolderRestService
 
 	/**
 	 * @ngdoc method
+	 * @name getPreparationActions
```
Comment (amaalej) [inline dataprep-webapp/src/app/services/folder/folder-service.js]: you meant getFolderActions

**Pull request merged**